dist/services/iiif-parser.js

```diff
@@ -118,13 +118,13 @@ function getChoiceItems(manifest) {
 function getLabelValue(label) {
   if (label && (0, _typeof2["default"])(label) === 'object') {
     // English
-    if (label.hasOwnProperty('@en')) {
-      return label['@en'].length > 0 ? label['@en'][0] : '';
+    if (label.hasOwnProperty('en')) {
+      return label['en'].length > 0 ? label['en'][0] : '';
     } // None
 
 
-    if (label.hasOwnProperty('@none')) {
-      return label['@none'].length > 0 ? label['@none'][0] : '';
+    if (label.hasOwnProperty('none')) {
+      return label['none'].length > 0 ? label['none'][0] : '';
```

Comment:
:+1: I didn't know the `@` was dropped for `none` as well but I see it in the specs.  Good catch!